### PR TITLE
Add note in vagrant docs about sudo

### PR DESCRIPTION
**Pull request by wallies**

### CODE REVIEW

**Comment by tayzlor** [comment]
can we not bake this into the scripts?

I'm also wondering a couple of things... Should we -

- Lock on minimum vagrant version https://docs.vagrantup.com/v2/vagrantfile/vagrant_version.html (i think most of us have been testing on 1.7.2?)
- Add an if around ```override.vm.provision :hosts``` to test if the vagrant-hosts plugin is installed
- Forward the ports for the web interfaces in the Vagrantfile (incase the vagrant-hosts plugin is disabled?)

**Comment by wallies** [comment]
Was going to bake it into the scripts, but wasnt sure. I'll add a minimum vagrant version of 1.7, as thats when things changed in the config.ssh. Will also add the hosts test as well

**Comment by tayzlor** [comment]
lets do min version and the if wrapper for now then and just a docs fix

**Comment by wallies** [comment]
Add check for vagrant and if wrapper. Didnt do port forwarding as we have the hosts on private network

File: docs/getting-started-guides/vagrant.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ export APOLLO_PROVIDER=vagrant
 sh bootstrap/apollo-launch.sh
 ```
 
+Note: You will either have to run the above command as sudo or have your hosts file writeable for the user running the command
```
Comment (tayzlor) [inline docs/getting-started-guides/vagrant.md]: does this doc only apply if using the vagrant-hosts plugin? might want to put a note in there about that if so.

**Comment by tayzlor** [comment]
Code here LGTM, docs look a bit suspect.

**Comment by asnaedae** [comment]
Aren't we already checking for if the plugin is installed, so isn't the check for the plugin during the rest of the vagrant file redundant?

  # throw error if vagrant-hosts not installed
  unless Vagrant.has_plugin?("vagrant-hosts")
    raise "vagrant-hosts plugin not installed"
  end

**Pull request merged**